.travis/runtox

```diff
@@ -16,6 +16,25 @@ ME=$(basename $0)
 SCRIPTDIR=$(readlink -f $(dirname $0))
 BANNERSIZE=90
 
+# LSR_MOLECULE_DOCKER_VERSION is the version string to pass
+# to pip install - this is usually a comparison operator
+# followed by a version number e.g. '<4.3' but in reality
+# may be anything that pip allows to follow a package name
+# in an install specification
+if [ -z "${LSR_MOLECULE_DOCKER_VERSION:-}" ]; then
+  if type docker > /dev/null 2>&1; then
+    DOCKER_SERVER_API_VERSION=$(docker version --format '{{.Server.APIVersion}}')
+    case "$DOCKER_SERVER_API_VERSION" in
+    1.39|1.[4-9]*) LSR_MOLECULE_DOCKER_VERSION="";;
+    1.3[0-8]|1.[0-2]*) LSR_MOLECULE_DOCKER_VERSION='<4.3';;
+    *) echo Error: unknown docker server api version "$DOCKER_SERVER_API_VERSION"; exit 1;;
```

Comment:
Not sure if it is useful to be this strict. Seems a special case for the <4.3 requirement could be neough.

Reply:
so something like this?
```bash
    if [ "$DOCKER_SERVER_API_VERSION" = "1.38" ]; then
      LSR_MOLECULE_DOCKER_VERSION='<4.3'
    else
      LSR_MOLECULE_DOCKER_VERSION=""
    fi
```
?